### ISSUE TITLE
[feat] Support Home Assistant Energy

### ISSUE THREAD

**Issue by bushvin** [zwave-js/zwavejs2mqtt#2051]
**Is your feature request related to a problem? Please describe.**
No, not really an issue.
Home Assistant Energy has been introduced in one of the latest HASS releases. It allows for tracking your energy generation and consumption without the need to create elaborate sensors, and whatnot.

**Describe the solution you'd like**
Apparently only devices that follow this guide, are supported to be added:
https://www.home-assistant.io/more-info/statistics/
https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

TL;DR: devices that produce state information using the kWh unit can be tracked, if they have a proper `state_class` and `device_class` assigned.

**Describe alternatives you've considered**
custom automations, sensors, etc...

**Additional context**
Not much to add here

**Comment by robertsLando** [comment]
@bushvin You want to introduce this on mqtt discovery?

**Comment by bushvin** [comment]
I have no idea if it has already been implemented through the js websocket. But yes, it would be nice to have on the MQTT discovery.

Unless there's another way this can be achieved? I wouldn't mind setting this manually.

**Comment by robertsLando** [comment]
@bushvin As I explained in docs I have no plan right now to keep mqtt discovery supported in future due to the lack of features it has and that zwavejs integration already covers that needs (and it's integrated on z2m with the zwavejs server). I would ask that question on zwavejs server or hass core repository

**Comment by bushvin** [comment]
ah, my apologies, I somehow missed that. Never mind me then.
